templates/menu.tpl

```diff
@@ -21,6 +21,17 @@
 {else}
 	<li><a target="_top" href="{#url_main#}">{$PALANG.pMenu_main}</a></li>
 {/if}
+{* list-server *}
+{if $authentication_has_role.global_admin}
+{strip}
+    <li><a target="_top" href="{#url_list_server#}">{$PALANG.pAdminMenu_list_server}</a>
```

Comment:
Not everybody will need support for multiple servers, so it would be nice to have a config option (maybe $CONF['multiple_servers']) that can disable this feature.

If disabled,
- the menu item should be hidden
- ServerHandler should refuse any edits, which means:
  - set _can_edit and _can_delete to Config::bool('multiple_servers') && $this->is_superadmin) in initStruct()
  - also add similar checks for can_create (in initMsg()) and delete()
- DomainHandler should not display the server field if multiple server handling is disabled